lib/src/main/java/graphql/nadel/NextgenEngine.kt

```diff
@@ -479,16 +482,13 @@ internal class NextgenEngine(
     ): Boolean {
         val topLevelField = topLevelFields.singleOrNull() ?: return false
 
-        if (topLevelField.fieldName == TypeNameMetaFieldDef.name) {
+        if (topLevelField.name == TypeNameMetaFieldDef.name) {
             return true
         }
-        val operationType = service.underlyingSchema.getTypeAs<GraphQLObjectType>(topLevelField.singleObjectTypeName)
-        val topLevelFieldDefinition = operationType.getField(topLevelField.name)
-        val isNamespacedLike = topLevelFieldDefinition?.arguments?.isEmpty() == true
```

Comment:
So the bug here is that the logic we use to identify "namespaced fields" is different here and in the internal introspection service.

This leads to a bug because this code thinks more fields can be handled by the internal introspection service, but in reality it can't so it fails.